proto/prost/private/tests/transitive_dependencies/transition.bzl

```diff
@@ -0,0 +1,32 @@
+load("//rust:defs.bzl", "rust_common")
+
+def _extra_toolchain_transition_impl(_settings, attr):
+    return {
+        "//command_line_option:extra_toolchains": [
+            attr.toolchain,
+        ],
+    }
+
+_extra_toolchain_transition = transition(
+    implementation = _extra_toolchain_transition_impl,
+    inputs = [],
```

Comment:
It seems like RBE needs previously defined extra toolchains? Might want to append your new toolchain here.

Reply:
Thanks!